### ISSUE TITLE
improve site accessibility features - aria labels

### ISSUE THREAD

**Issue by okjintao** [Badger-Finance/v2-ui#154]
Accessibility

Screen readers will not be able to identify several buttons due to a lack of accessible naming. Buttons without an explicit text label or an aria-label property are announced as ‘button’ by screen readers.

Potential improvements

Addition of appropriate labeling and/or an aria-label property to all buttons.
